duckduckgo/Android
# Restrict iar parameters

Task/Issue URL: https://app.asana.com/0/414730916066338/1200465880706413

### Description
This PR validates that a vertical is a major vertical before being added to the iar parameter

### Steps to test this PR

_IAR added only if major_
1. Search for lol
1. Click on the videos tab
1. In the logcat filter by `iar=videos`, nothing should show up.
1. Select the search bar and search for `test`
1. The logcat should now show results
1. Filter the logcat by `ia=currency`
1. Search `usd to gbp`
1. The logcat should return several results
1. Filter the logcat by `iar=currency`
1. Select the search bar and search for `test`
1. The logcat should not show any results
